x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/agent_policy/components/agent_policy_advanced_fields/hooks.tsx

```diff
@@ -202,7 +205,7 @@ function getDefaultDownloadSource(
       }),
       defaultDownloadSourceDisabledMessage
     ),
-    value: DEFAULT_SELECT_VALUE,
+    value: DEFAULT_DOWNLOAD_SOURCE_ID,
```

Comment:
I think the fix here should be more like the select for outputs work https://github.com/elastic/kibana/blob/main/x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/agent_policy/components/agent_policy_advanced_fields/index.tsx#L658-L669 

Correct me if I am wrong @criamico but I think binary source work the same as output, where a user can put:
* null => this will use the default binary source and if the default binary source change it will update policies
* a regular binary source policy id 
* the default binary source id and in that case it will not affect the policy  if the default binary source change

Reply:
in that case should we have both options?
* Default (currently source 1)
* source 1

it's what we are doing for outputs

Reply:
oh okay, probably we can do that

Reply:
Yes , the binary source works the same way as outputs where `download_source_id` can be either null or a specific id (default or otherwise).

Reply:
updated, now the default option is visible next to the default one, did the same for Fleet server hosts

<img width="1505" alt="image" src="https://github.com/user-attachments/assets/931df21b-066e-451c-8b83-e6c767b5925f" />